### DISCUSSION TITLE
I created a plugin to turn Figma designs into Tailwind React code.

### DISCUSSION THREAD

**Discussion by loweceo** [tailwindlabs/tailwindcss#12017] [Show and tell]
![Capture d'écran 2023-09-16 003340](https://github.com/tailwindlabs/tailwindcss/assets/112697062/b4c70bac-9dae-4492-9849-b78bd6073129)

I created a Figma plugin to convert designs into Tailwind React code.

I haven't had any feedback on the plugin yet, can you give me some to improve my plugin?

Thank!

https://www.figma.com/community/plugin/1283055580669946018/Figma-to-Tailwind-React-Code---by-Pagesloft-(ALPHA)

**Comment by GuillaumeEtendard** [comment]
Great plugin, the code that has been generated was very nice
